tasks/lzards-backup/src/index.ts

```diff
@@ -355,7 +385,7 @@ export const backupGranulesToLzards = async (event: HandlerEvent) => {
 };
 
 export const handler = async (
-  event: CumulusMessage | CumulusRemoteMessage,
+  event: CumulusMessage | CumulusRemoteMessage | any,
```

Comment:
What's up with this? Doesn't this invalidate typing `event` with the first two options if it can be `any`?

Reply:
Yeah I added this type in an attempt to figure out my CMA issues but the issue turned out to be schema configuration errors rather than the lambda having issues 😓